frb_codegen/src/library/commands/cargo_expand.rs

```diff
@@ -166,4 +167,29 @@ mod another {}";
         let extracted = extract_module(src, Some(String::from("another"))).unwrap();
         assert_eq!(String::from(""), extracted);
     }
+
+    #[test]
+    pub fn test_extract_module_with_prefix() {
+        let src = "pub mod parent {
+    mod another {
+        // some code
+    }
+}";
+        let extracted = extract_module(src, Some(String::from("another"))).unwrap();
```

Comment:
nit: it seems that we do not have `another` module (but only parent::another). I wonder whether we should fail, instead of outputing the whole text?

Reply:
![image](https://github.com/fzyzcjy/flutter_rust_bridge/assets/59032810/474ec21b-d54b-4058-9d23-a9fce6c2d724)

I just mimic the old behavior because I don't know if other code depends on it. :)

Reply:
I see. If CI is green, then I guess it is OK. Since that does not seem to be the correct behavior, I guess we can fix it.

Reply:
Great. Just return "" ? It looks like `extract_module` never fail.

Reply:
I personally suggest making it fail (return `anyhow::Result` and use `bail!("error message")`), but panic may be OK as well